README.md

```diff
@@ -20,9 +20,31 @@ WIKI: https://wiki.mozilla.org/Idea-Town
 
 * [Install Docker](http://docs.docker.com/mac/started/) and docker-compose
 
-* `docker up`
+* Make sure you have a default Docker machine, i.e.:
+  * `docker-machine create -driver virtualbox default`
 
-* Visit http://0.0.0.0:8000
+* Make sure your shell can see the default Docker machine, i.e.:
+  * `eval "$(docker-machine env default)"`
+
+* Create & set up the Docker containers:
+  * `docker-compose up`
+  * This may take some time.
+
+* To see the IP address of the default Docker machine:
+  * `docker ip default` 
```

Comment:
This is great, thanks for including some docs!

For some reason, on my machine, `docker-machine ip default` works, but `docker ip default` doesn't.